tekton/src/params/install.yml.j2

```diff
@@ -92,6 +92,33 @@
   description: Set to 'system' if using the shared system JDBC configuration, or 'workspace-application' if using a dedicated JDBC configuration.
   default: ""
 
+# Dependencies - JDBC settings
+- name: configure_external_db
```

Comment:
Minor thing... add:

"# -----------------------------------------------------------------------------"

To keep the same standard

Reply:
added line